shoop/themes/classic_gray/templates/classic_gray/shoop/front/macros.jinja

```diff
@@ -34,8 +34,8 @@
     </div>
 {% endmacro %}
 
-{% macro render_cross_sell_products(product, relation_type="", title="") %}
-    {%- set cross_sell_products = shoop.product.get_product_cross_sells(product, relation_type) %}
+{% macro render_cross_sell_products(product, relation_type="", title="", count=4) %}
```

Comment:
I don't think that render_croll_sell_products should be here. It should probably be under cross_sells_plugin.jinja.